INSTALL

```diff
@@ -326,6 +326,11 @@
                    Don't build support for datagram based BIOs. Selecting this
                    option will also force the disabling of DTLS.
 
+  enable-devcryptoeng
+                   Build the /dev/crypto engine.  It is automatically selected
+		   on BSD implementations, in which case it can be disabled with
```

Comment:
What happened with the indentation here?

Reply:
Tabs vs spaces, sorry.  Fixed.